desks-front/src/components/UI/Desk/DeskContainer/DeskContainer.jsx

```diff
@@ -1,89 +1,87 @@
 import { useEffect, useState } from "react";
-
 import { Button, ButtonToolbar, OverlayTrigger } from "react-bootstrap";
 import "./DeskContainer.css";
 import { AvailableTooltip } from "../../Tooltip/Tooltips";
 import DropdownMenu from "../../DropdownMenu";
 import SelectedDeskDisplay from "../SelectedDeskDisplay";
-
-// temporary data, should be fetched from firebase
-const desks = [
-    { location: "hd", room: 1, desk: 1, addedBy: "admin@admin.com" },
-    { location: "ns", room: 1, desk: 1, addedBy: "admin@admin.com" },
-    { location: "ns", room: 1, desk: 5, addedBy: "admin@admin.com" },
-    { location: "ns", room: 1, desk: 2, addedBy: "admin@admin.com" },
-    { location: "md", room: 1, desk: 5, addedBy: "admin@admin.com" },
-    { location: "md", room: 2, desk: 2, addedBy: "admin@admin.com" },
-];
+import getDesksByLocationFromDb from "../../../../utils/db/getDesksByLocationFromDb";
 
 const menuOptions = [
```

Comment:
I made a Jira ticket `[JSSBG-29]` for things like this, you see, you only changed the formatting here (and it was probably done by Prettier anyway) and the result is cluttering of the diff file here, we don't want that - hence the abstraction, check out the ticket (either @stamenione or @NikolaDmitrasinovic).